File: Task-03/Sub-Task 3/Ruby/Ques3.rb
n = 5

(1..n).step(2) do |i|
  puts ' ' * ((n - i) / 2) + '*' * i
end

(n-2).step(1, -2) do |i|
  puts ' ' * ((n - i) / 2) + '*' * i
end

File: Task-03/Sub-Task 3/Elixir/Ques3.ex
n = 5

for i <- 1..n, rem(i, 2) == 1 do
  IO.puts String.duplicate(" ", div(n - i, 2)) <> String.duplicate("*", i)
end

for i <- (n-2)..1, rem(i, 2) == 1 do
  IO.puts String.duplicate(" ", div(n - i, 2)) <> String.duplicate("*", i)
end
